RecoEgamma/ElectronIdentification/python/FWLite.py

```diff
@@ -10,16 +10,32 @@
 # Usage example in RecoEgamma/ElectronIdentification/test
 
 class ElectronCutBasedID(object):
-    """ Electron cut based ID wrapper class.
+    """ Electron cut based ID wrapper class. Allows testing cut based ID working points
+    with python.
     """
     def __init__(self, name, tag, working_points):
         self.name = name 
         self.tag = tag
         self.working_points = working_points
 
     def passed(self, ele, rho, wp):
-        '''return true if ele passes wp
-        see https://twiki.cern.ch/twiki/bin/viewauth/CMS/CutBasedElectronIdentificationRun2#Offline_selection_criteria_for_V'''
+        '''return true if the electron passes the cut based ID working point.
```

Comment:
it seems to me that rho is not used in this function. why did you keep it in the argument list? 
can you remove it ?

Reply:
It is, see [line 54](https://github.com/cbernet/cmg-cmssw/blob/338f24056b272e11b2b8b0a844934b76afa85c77/RecoEgamma/ElectronIdentification/python/FWLite.py#L54) for example.